crates/transaction-pool/src/blobstore/disk.rs

```diff
@@ -110,6 +121,7 @@ struct DiskFileBlobStoreInner {
     blob_cache: Mutex<LruMap<TxHash, BlobTransactionSidecar, ByLength>>,
     size_tracker: BlobStoreSize,
     file_lock: RwLock<()>,
+    files_to_delete: RwLock<HashSet<B256>>,
```

Comment:
This field is used to hold the txhashes to delete, maybe we name it as `txs_to_delete` or something else, the `files_to_delete` is ambiguity

Reply:
Done!